src/validations/rules/sar.sch

```diff
@@ -176,6 +176,84 @@
 
     </sch:pattern>
 
+    <sch:pattern
+        id="results">
+        <sch:rule
+            context="oscal:observation">
```

Comment:
How is this context (and subordinate assertions) constrained to automated tool observations and findings?

Reply:
My reading of the Guide Section 4.7 was that type='finding' and method='test' indicated an automated test.  Perhaps that was reading between the lines to much?

Reply:
Section 4.7 uses `observation/origin/actor/@type eq 'tool'` to identify observations originating from automated tools.
Section 4.7 also requires (or at least attempts to require) that the target of `observation/origin/actor/@tactor-uuid` was copied from the SAP into somewhere other than the non-existent `assets` (perhaps `parent::result/local-definitions/assessment-assets/component`). See also https://github.com/GSA/fedramp-automation-guides/issues/49.